frontend/src/components/common/Profile.tsx

```diff
@@ -0,0 +1,43 @@
+import UserProfile from "../user.png";
+import users from "../../dummydata";
+
+interface Users {
+  id: number;
+  first_name: string;
+  last_name: string;
+  category: string;
+  total_items: string;
+  score: string;
+  taken_category: number;
+}
+
+const Profile = () => {
+  return (
+    <div className="p-[4rem] ml-[5rem] flex flex-col-2 items-center">
+      <div>
+        <h1 className="p-[1rem] text-[26px] font-bold tracking-normal">
+          Dashboard
+        </h1>
+        <img
+          src={UserProfile}
+          alt="profile"
+          className="h-[150px] bg-gray-200 p-2 rounded"
+        />
+      </div>
+      {users.slice(1, 2).map((user: Users, id: number) => (
```

Comment:
You're just renaming the `index` of the callback rather than using the `id` of the user.

At **line:28** please use `key={user.id}` instead.